C++ API parity: Softplus

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27309 C++ API parity: MultiheadAttention
* #27382 C++ API parity: Linear
* #27538 C++ API parity: Threshold
* #27537 C++ API parity: Tanhshrink
* #27536 C++ API parity: Tanh
* #27535 C++ API parity: Softsign
* #27534 C++ API parity: Softshrink
* **#27489 C++ API parity: Softplus**

Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

pbelevich committed:
C++ API parity: Softplus

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"




[ghstack-poisoned]

yf225 reviewed (approved):
Thanks for the awesome work @pbelevich!

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softplus"


Differential Revision: [D17835410](https://our.internmc.facebook.com/intern/diff/D17835410)

[ghstack-poisoned]
